### ISSUE TITLE
@Param in JSDoc gives sometimes missing whitespace error

### ISSUE THREAD

**Issue by flemgs** [palantir/tslint#3304]
### Bug Report

- __TSLint version__: 5.5.0
- __TypeScript version__: 2.5.2
- __Running TSLint via__: (pick one) CLI / Node.js API / 

#### TypeScript code being linted

```ts
/**
 * functionDescription
 * @param myParam
 */
```

with `tslint.json` configuration:

```json
"whitespace": [
      true,
      "check-separator"
    ],
```

#### Actual behavior
Since migration from typescript 2.4 to 2.5, TSLint rule 'check-separator' in 'whitespace' give errors on JSDoc comments with this format : 
/**
* functionDescription
* @param myParam
*/
It says that there is a missing whitespace between '@param' and 'myParam' when it is actually present. It seems to be random since it does not concern all JSDoc comments with '@param' in it.

#### Expected behavior
TSlint should not give an error

**Comment by ajafff** [comment]
Duplicate of #3251
